apps/docs/theme.config.tsx

```diff
@@ -13,6 +13,16 @@ const config: DocsThemeConfig = {
   footer: {
     text: "OpenStatus Documentation",
   },
+  useNextSeoProps() {
+    return {
+      titleTemplate: "%s | OpenStatus Documentation",
+      openGraph: {
+        url: "https://openstatus.dev/api/og",
+        title: "OpenStatus Documentation",
+        description: "An Open Source alternative for your next Status Page",
+      },
```

Comment:
Nice, nextra ftw! Always wanted to use nextra in general. 

I'd just only add the `description` and the `twitter` props to it. Otherwise lgtm! 🚀

```suggestion
      titleTemplate: "%s | OpenStatus Documentation",
      description: "An Open Source alternative for your next Status Page",
      openGraph: {
        url: "https://openstatus.dev/api/og",
        title: "OpenStatus Documentation",
        description: "An Open Source alternative for your next Status Page",
      },
      twitter: {
        cardType: 'summary_large_image',
        site: 'https://nextra.vercel.app'
      },
```